packages/elements/src/components/ino-nav-drawer/ino-nav-drawer.tsx

```diff
@@ -67,6 +67,10 @@ export class NavDrawer implements ComponentInterface {
       this.drawerInstance.open = this.open || false;
     }
 
+    this.el.shadowRoot
+      .querySelector('.mdc-drawer')
+      .addEventListener('MDCDrawer:closed', this.closeDrawer);
```

Comment:
We need to remove the listener on unloading.

Reply:
Done ✅ I don't know why I always forget to remove the event listener...